dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/MetadataDumperTest.java

```diff
@@ -16,19 +16,150 @@
  */
 package com.google.edwmigration.dumper.application.dumper;
 
+import com.google.edwmigration.dumper.application.dumper.connector.Connector;
 import com.google.edwmigration.dumper.application.dumper.connector.bigquery.BigQueryLogsConnector;
+import com.google.edwmigration.dumper.application.dumper.connector.hive.HiveMetadataConnector;
+import java.io.File;
+import org.junit.After;
+import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/** @author shevek */
+/**
+ * @author shevek
+ */
 @RunWith(JUnit4.class)
 public class MetadataDumperTest {
 
+  private File file;
+
+  @After
+  public void tearDown() {
+    if (file != null && file.exists()) file.delete();
+  }
+
   @Test
   public void testInstantiate() throws Exception {
     MetadataDumper dumper = new MetadataDumper();
-    // dumper.run("--help");    // Calls System.exit(1)
     dumper.run("--connector", new BigQueryLogsConnector().getName(), "--dry-run");
   }
+
+  @Test
+  public void testCreatesDefaultOutputZip() throws Exception {
+    // Arrange
+    MetadataDumper dumper = new MetadataDumper().withExitOnError(false);
```

Comment:
I would recommend moving this line out of the test methods to a field. In `testInstantiate()` it can be overwritten with `dumper = new MetadataDumper();` if the `.withExitOnError(false)` is not required there.